features/facebookImport/src/components/dataOverviewMiniStory/dataOverviewMiniStory.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import DataBubblesAll from "../dataViz/dataBubblesAll.jsx";
+
+import i18n from "../../i18n.js";
+
+import "./dataOverviewMiniStory.css";
+
+const DataOverviewMiniStory = ({ data }) => {
+    data.sort(function (a, b) {
+        return b.count - a.count;
+    });
+
+    const dataBubblesWidth = 400;
+    const dataBubblesHeight = 400;
```

Comment:
maybe bubbleVizHeight? The bubbles do not actually have that width and height but the visualisation in general, while the color is actually the bubbles color. Would be a little confusing i think